Resources/Locale/en-US/_NF/reagents/meta/chemicals.ftl

```diff
@@ -0,0 +1,2 @@
+reagent-name-salicylicacid = Salicylic Acid
```

Comment:
```suggestion
reagent-name-salicylicacid = salicylic acid
```
For consistency with other entries in the guidebook.  Missed this at first, sorry.